apidef/api_definitions.go

```diff
@@ -477,6 +477,8 @@ type AuthConfig struct {
 type SignatureConfig struct {
 	Algorithm        string `mapstructure:"algorithm" bson:"algorithm" json:"algorithm"`
 	Header           string `mapstructure:"header" bson:"header" json:"header"`
+	UseParam         bool   `mapstructure:"use_param" bson:"use_param" json:"use_param"`
+	Param            string `mapstructure:"param" bson:"param" json:"param"`
```

Comment:
Lets make it consistent with another config options we have https://github.com/TykTechnologies/tyk/blob/76103a37c1a082c167d085bfee5d8fdde378d0ab/apidef/api_definitions.go#L468

Just name it ParamName  `param_name`.

Reply:
As the header parameter is just `header` and not `header_name`, I was following that naming. But you are right to make it equal to the `param_name`.